pkg/client/jobs.go

```diff
@@ -0,0 +1,77 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	apiv1 "github.com/acorn-io/runtime/pkg/apis/api.acorn.io/v1"
+	"github.com/acorn-io/runtime/pkg/labels"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8slabels "k8s.io/apimachinery/pkg/labels"
+	kclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func (c *DefaultClient) JobGet(ctx context.Context, name string) (*apiv1.Job, error) {
+	job := &apiv1.Job{}
+	return job, c.Client.Get(ctx, kclient.ObjectKey{
+		Name:      name,
+		Namespace: c.Namespace,
+	}, job)
+}
+
+func (c *DefaultClient) JobList(ctx context.Context, opts *JobListOptions) ([]apiv1.Job, error) {
+	result := &apiv1.JobList{}
+
+	selector, err := k8slabels.Parse(fmt.Sprintf("%s=%s", labels.AcornManaged, "true"))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Is this label selector necessary? We are querying for an Acorn API object, so I don't think any of them should even exist without this label on them.

Reply:
This is some leftover code, thanks for catching it. I had a previous implementation where the client just got the underlying `batchv1.Job` object. However, that no logic longer exists and I will update this code accordingly.